internal/parser.go

```diff
@@ -155,13 +153,12 @@ func (p *Parser) parseGlobal() error {
 		return err
 	}
 
-	re := regexp.MustCompile(osCommandRegexp)
 	for k, v := range g.Shared.Environment {
-		_, cmd := p.parseSystemCmd(re, v)
+		_, cmd := p.parseSystemCmd(osCommandRegexp, v)
 
 		if cmd == "" {
 			g.Shared.Environment[k] = v
-			os.Setenv(k, v)
+			_ = os.Setenv(k, v)
```

Comment:
I don't think this is necessary. What do you think?

Reply:
i usually do this to stop the linters from complaining that we're not handling errors